package.json

```diff
@@ -24,6 +24,7 @@
   ],
   "license": "SEE LICENSE IN LICENSE.txt",
   "dependencies": {
+    "tcp-port-used": "^0.1.2",
```

Comment:
Will this work if we update the node we use to run the DA to a newer version? see: https://github.com/stdarg/tcp-port-used/issues/3

Reply:
Yeah looks like this one has an issue with node> 7.7. Do we have any plans to update the node version we use? I am sure I can find other packages to do the same thing, but just wanna check if we have something on the agenda.

Reply:
I changed the package none the less :) Thanks for catching it.